net/nimble/controller/src/ble_ll_conn_priv.h

```diff
@@ -139,7 +139,7 @@ uint32_t ble_ll_conn_get_ce_end_time(void);
 void ble_ll_conn_event_halt(void);
 uint8_t ble_ll_conn_calc_used_chans(uint8_t *chmap);
 void ble_ll_conn_reset_pending_aux_conn_rsp(void);
-
+bool ble_ll_conn_init_aux_conn_is_sent(void);
```

Comment:
ble_ll_conn_init_is_aux_conn_req_sent would be better
or we already have 'pending_aux_conn_rsp' as above so let's keep it one way or another for both